src/components/SimMenu.tsx

```diff
@@ -265,7 +277,7 @@ class SimMenu extends React.PureComponent<Props, State> {
           style={{ borderLeft: `1px solid ${theme.borderColor}` }}
         >
           <ItemIcon icon={faPlay} />
-          Run
+          {LocalizedString.lookup(tr('Run', 'Begin program execution'), locale)}
         </RunItem>
       );
 
```

Comment:
github won't let me comment on unchanged lines, but "Layout" and "World" below need `tr()`

Reply:
and "Reset World"